dev-tools/reconfigurator-cli/tests/output/cmds-nexus-generation-autobump-stdout

```diff
@@ -0,0 +1,2162 @@
+INFO some zones not yet up-to-date, zones_currently_updating: [ZonePropagationIncomplete { zone_id: 4c6a45f3-6754-4082-a7dd-11d59b0127b1 (service), zone_kind: Nexus, reason: MissingInInventory { bp_image_source: Artifact { version: Available { version: ArtifactVersion("1.0.0") }, hash: ArtifactHash("0e32b4a3e5d3668bb1d6a16fb06b74dc60b973fa479dcee0aae3adbb52bf1388") } } }, ZonePropagationIncomplete { zone_id: 9882b9e4-9a10-48ef-bf78-0cc1317dadfe (service), zone_kind: Nexus, reason: MissingInInventory { bp_image_source: Artifact { version: Available { version: ArtifactVersion("1.0.0") }, hash: ArtifactHash("0e32b4a3e5d3668bb1d6a16fb06b74dc60b973fa479dcee0aae3adbb52bf1388") } } }, ZonePropagationIncomplete { zone_id: a40e7821-3ec7-4aa9-995d-7d26ecec7d0f (service), zone_kind: Nexus, reason: MissingInInventory { bp_image_source: Artifact { version: Available { version: ArtifactVersion("1.0.0") }, hash: ArtifactHash("0e32b4a3e5d3668bb1d6a16fb06b74dc60b973fa479dcee0aae3adbb52bf1388") } } }]
+generated blueprint a5a8f242-ffa5-473c-8efd-2acf2dc0b736 based on parent blueprint d60afc57-f15d-476c-bd0f-b1071e2bb976
+blueprint source: planner with report:
+planning report:
+* skipping noop zone image source check on sled 2b8f0cb3-0295-4b3c-bc58-4fe88b57112c: all 10 zones are already from artifacts
+* skipping noop zone image source check on sled 98e6b7c2-2efa-41ca-b20a-0a4d61102fe6: all 9 zones are already from artifacts
+* skipping noop zone image source check on sled d81c6a84-79b8-4958-ae41-ea46c9b19763: all 9 zones are already from artifacts
+* zone updates waiting on zone propagation to inventory
+* waiting to update top-level nexus_generation: new Nexus zones are not in inventory yet
+
+
+
+
+> # If we're missing the "Active" records, we won't be able to lookup the
+> # current Nexus image, which will cause planning to fail
+> set active-nexus-zones
+set 0 explicit active Nexus zones
+
+> blueprint-plan latest latest
+error: failed to construct planning input: no Nexus zones found at current active generation (1)
```

Comment:
Technically this output is coming from the reconfigurator-cli, but, even if it wasn't, the planner would throw a similar error.